templates/contribute/style.md

```diff
@@ -314,6 +314,15 @@ is preferred over
 example : ∀ (n : ℕ), 0 ≤ n := Nat.zero_le
 ```
 
+Note that pattern-matching does not count as the proof starting by introducing variables.
+For example, the following is a valid use case of having an hypothesis right of the column:
```

Comment:
```suggestion
For example, the following is a valid use case of having a hypothesis right of the column:
```
I think that since the `h` is not silent, `a` should be used here.